conda_index/index/sqlitecache.py

```diff
@@ -117,7 +133,7 @@ def db(self) -> sqlite3.Connection:
         """
         Connection to our sqlite3 database.
         """
-        conn = common.connect(self.db_filename)
+        conn = common.connect(str(self.db_filename))
```

Comment:
I thought it was neat that sqlite3 also accepts Path, but the DBAPI signature is str.